datasets/urdu_fake_news/README.md

```diff
@@ -0,0 +1,140 @@
+---
+annotations_creators:
+  - expert-generated
+language_creators:
+  - expert-generated
+languages:
+  - ur
+licenses: []
```

Comment:
is there no license or is the license unknown ?
If it's unknown please use 
```
licenses:
- unknown
```

Reply:
made the change